seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-1/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMysqlMultipleTablesIT.java

```diff
@@ -171,7 +171,7 @@ public void testMysqlJdbcSingleTableE2e(TestContainer container)
 
     @DisabledOnContainer(
             value = {},
-            type = {EngineType.SPARK, EngineType.FLINK},
+            type = {EngineType.FLINK},
```

Comment:
check all e2e testcase and remove EngineType.SPARK

Reply:
@Carl-Zhou-CN

Reply:
ok